modules/oxide/src/less/theme/content/accessibility-checker/accessibility-checker.less

```diff
@@ -4,6 +4,10 @@
 @accessibility-overlay-opacity-90: 90%;
 @accessibility-overlay-opacity-60: 60%;
 
+@accessibility-box-shadow-color: #222f3e80;
```

Comment:
Should this be configurable for dark mode?

Reply:
in dark mode this box shadow will also dim the rest of the content and draw attention to the focused element, there is perhaps some improvement we can do on this, I'll discuss with product.

Reply:
dark mode in general is looking good, there are other dark mode specific issues that have some effects on other plugins needs to addressed.